<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="Masque">
      <property name="text">
       <string>MasqueüëÅÔ∏è:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="maskInput">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Adresse IPü§≥</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ipInput">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#ff55ff;&quot;&gt;A&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="calculateButton">
      <property name="text">
       <string>Calculer les sous-r√©seaux</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="resultList">
      <item>
       <property name="text">
        <string>Sous-r√©seaux de l'Adresse IP Entr√©:</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignBottom</set>
       </property>
       <property name="background">
        <brush brushstyle="Dense7Pattern">
         <color alpha="255">
          <red>255</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
       <property name="foreground">
        <brush brushstyle="DiagCrossPattern">
         <color alpha="255">
          <red>85</red>
          <green>0</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::FormatTextDirectionLtr"/>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionMasque"/>
    <addaction name="actionAdresse_IP"/>
    <addaction name="actionSous_r_seaux"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionMasque">
   <property name="text">
    <string>Masque</string>
   </property>
  </action>
  <action name="actionAdresse_IP">
   <property name="text">
    <string>Adresse IP</string>
   </property>
  </action>
  <action name="actionSous_r_seaux">
   <property name="text">
    <string>Sous-r√©seaux</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>calculateButton</tabstop>
  <tabstop>resultList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
